.dockerignore

```diff
@@ -0,0 +1,26 @@
+# we try to aoid adding files to the docker images that change often
+# or that are not needed for running the docker image
+# tis greatly reduces the amount of times we need to rerun `npm install` when building image locally
+# https://codefresh.io/blog/not-ignore-dockerignore/
+# https://docs.docker.com/engine/reference/builder/#dockerignore-file
+
+# consider them hidden
+.*
```

Comment:
I see, before I came to this entry Ih had like 15 entries starting with a dot in the file,
then I read the article I mention in the file and changed it to this.

You can always exclude certain files like in gitignore:
`!.babelrc` 
Then it becomes a white list, instead of blacklisting every single "hidden" file. :)